puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh

```diff
@@ -47,9 +47,6 @@ bundle exec rake db:drop db:create db:migrate
 # Now let's introduce the plugin
 echo "gem '${plugin_name}', :path => '${PLUGIN_ROOT}'" >> bundler.d/Gemfile.local.rb
 
-# Plugin specifics..
-[ -e ${PLUGIN_ROOT}/script/ci/katello.yml ] && cp ${PLUGIN_ROOT}/script/ci/katello.yml ${PLUGIN_ROOT}/config/katello.yml
```

Comment:
AFAIK, we don't need this for katello. I'm actually trying to remove script/ci/katello.yml from our repo which prompted this PR.